src/main/java/uk/ac/ebi/ampt2d/metadata/rest/controllers/StudyRestController.java

```diff
@@ -69,6 +70,37 @@
         return ResponseEntity.ok(resources);
     }
 
+    @ApiOperation(value = "Get the latest version of a study based on accession ")
+    @ApiParam(name = "accession", value = "Study's accession", type = "string", required = true, example = "EGAS0001")
+    @RequestMapping(method = RequestMethod.GET, path = "search/accession", produces = "application/json")
+    @ResponseBody
+    @SuppressWarnings("unchecked")
+    public ResponseEntity<Resources<StudyResource>> findStudiesByAccession(String accession) {
+        List<Study> studies = studyService.findStudiesByAccession(accession);
+
+        Resources<StudyResource> resources = (Resources<StudyResource>) resourceAssembler.toResources(Study.class, studies);
+
+        return ResponseEntity.ok(resources);
+    }
+
+    @ApiOperation(value = "Get the list of studies filtered by release date")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "from", value = "starting date", dataType = "string", format = "date", paramType = "query", example = "2016-01-01"),
+            @ApiImplicitParam(name = "to", value = "ending date", dataType = "string", format = "date", paramType = "query", example = "2018-01-01")
+    })
+    @RequestMapping(method = RequestMethod.GET, path = "search/release-date", produces = "application/json")
+    @ResponseBody
+    @SuppressWarnings("unchecked")
+    public ResponseEntity<Resources<StudyResource>> findStudiesByReleaseDate(String from, String to) {
```

Comment:
Please check if there is any way to use LocalDate instead of String. Otherwise, if the dates can't be parsed the user should receive an error, not an OK.

Reply:
Yes, I have made it to use LocalDate. But it needs to be annotated as a @RequestParam with a @DateTimeFormat. If a user input can not be parsed into a LocalDate objects, a MethodArgumentTypeMismatchException will be thrown and handled by an @ExceptionHandler.